From d103028833ac24d98319e637422fa90f33020911 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Fri, 14 Oct 2016 15:02:17 -0400
Subject: [PATCH 1240/1722] drm/amdgpu/dal: fix warning

I forgot to fix this up when I rebased by reset fix patch
on amd-staging-4.7 and added DAL support.

Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm.c
index 6c4636b..a022e60 100644
--- a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm.c
@@ -136,8 +136,6 @@ static bool dm_check_soft_reset(void *handle)
 
 static int dm_soft_reset(void *handle)
 {
-	struct amdgpu_device *adev = (struct amdgpu_device *)handle;
-
 	/* XXX todo */
 	return 0;
 }
-- 
2.7.4

